Features/Users/users.feature

```diff
@@ -67,3 +67,10 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate that users have the "Delete" button available
+        Given I am on the Users page
```

Comment:
The objective of the scenario is not to view a users' data, but to be on a user's page, so 'view' might mislead the reader.
But yes, we might shorten the sentences. Suggestion:
```Given I am on a user's page```
```Then I should see a "Delete" button in the action bar```